e2e/baseFixtures.js

```diff
@@ -138,11 +138,19 @@ exports.test = base.test.extend({
      * Extends the base page class to enable console log error detection.
      * @see {@link https://github.com/microsoft/playwright/discussions/11690 Github Discussion}
      */
-    page: async ({ page, failOnConsoleError }, use) => {
+    page: async ({ page, failOnConsoleError }, use, testInfo) => {
```

Comment:
We should move this into appActions. The baseFixtures fixture is designed to disappear as native functionality is added to the `@playwright/test` or plugin ecosystem.

In this case, the testnotes function is specific to open mct